source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/imu/imu.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+import torch
+from collections.abc import Sequence
+from typing import TYPE_CHECKING
+
+import omni.physics.tensors.impl.api as physx
+from pxr import UsdPhysics
+
+import omni.isaac.lab.sim as sim_utils
+import omni.isaac.lab.utils.math as math_utils
+from omni.isaac.lab.markers import VisualizationMarkers
+
+from ..camera.utils import convert_orientation_convention, create_rotation_matrix_from_view
+from ..sensor_base import SensorBase
+from .imu_data import IMUData
+
+if TYPE_CHECKING:
+    from .imu_cfg import IMUCfg
+
+
+class IMU(SensorBase):
+    """The inertia measurement unit."""
```

Comment:
Would add a better description on IMU sensor and this class's functionality and what users can expect out of it.